tasks/bundle-js.js

```diff
@@ -28,6 +30,19 @@ function patchFirefoxJS(/** @type {string} */code) {
     return code;
 }
 
+function patchMV3JS(/** @type {string} */code) {
+    // MV3 moves a few APIs around
+    code = replace(code, 'chrome.browserAction.setIcon', 'chrome.action.setIcon');
+    code = replace(code, 'chrome.browserAction.setBadgeBackgroundColor', 'chrome.action.setBadgeBackgroundColor');
+    code = replace(code, 'chrome.browserAction.setBadgeText', 'chrome.action.setBadgeText');
+
+    // Background serivice workers do not have access to "foreground" APIs like chrome.i18n
+    code = replace(code, 'chrome.i18n.getUILanguage()', "navigator.language.split('-')[0]");
+    // TODO: Fix me
+    code = replace(code, 'window.matchMedia(query).matches', 'false');
```

Comment:
There are multiple ways to implement CSS Media queries:
 - use content script context
 - spawn a "sandbox" which has an invisible window. This is the cleanest and the most reliable way, but spawning an entire process just to query a boolean value seems wasteful...

Reply:
Then try doing it in context script

Reply:
Done!